lib/engine/game/g_18_zoo/game.rb

```diff
@@ -1125,6 +1129,10 @@ def it_s_all_greek_to_me
           @it_s_all_greek_to_me ||= company_by_id('IT_S_ALL_GREEK_TO_ME')
         end
 
+        def it_s_all_greek_to_me_active?
```

Comment:
it_is

Reply:
what do you mean? the power name is "IT'S ALL GREEK TO ME"
maybe "it_s_all_greek_to_me_is_active?"

Reply:
the power name might be, but it_s reads badly, it_is_all_greek_to_me_active? is readable

Reply:
I can simplify to greek_to_me too

Reply:
done in f3d4a846